packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -68,19 +68,23 @@ export interface RichTextSendBoxOptions extends RichTextEditorOptions {
   /**
    * Optional callback to handle an inline image that's inserted in the rich text editor.
    * When not provided, pasting images into rich text editor will be disabled.
+   * @param imageAttributes - attributes of the image such as id, src, style, etc.
```

Comment:
It looks like  `RichTextSendBoxOptions` should part of the composite package, not components